server/app/views/applicant/ProgramIndexView.java

```diff
@@ -121,12 +121,74 @@ public Content render(
         request, personalInfo, messages, bundle, /* includeAdminLogin= */ true, applicantId);
   }
 
+  public Content renderWithoutApplicant(
+      Messages messages,
+      Http.Request request,
+      ApplicantService.ApplicationPrograms applicationPrograms
+      //   ImmutableList<String> selectedCategoriesFromParams,
+      ) {
+    HtmlBundle bundle = layout.getBundle(request);
+    bundle.setTitle(messages.at(MessageKey.CONTENT_FIND_PROGRAMS.getKeyName()));
+    bundle.addMainContent(topContent(request, messages, Optional.empty()), div("no cookies!"));
+    // bundle.addMainContent(
+    //       topContent(request, messages, Optional.empty()),
+    //       mainContent(
+    //           request,
+    //           messages,
+    //           personalInfo,
+    //           applicationPrograms,
+    //           applicantId,
+    //           messages.lang().toLocale(),
+    //           bundle,
+    //           profile));
+    return layout.render(bundle);
+  }
+
+//   private DivTag mainContentNoApplicant(
+//       ApplicantService.ApplicationPrograms relevantPrograms,
+//       Messages messages,
+//       Http.Request request) {
+//     DivTag content =
+//         div().withId("main-content").withClasses(ApplicantStyles.PROGRAM_CARDS_PARENT_CONTAINER);
+
+//     // The different program card containers should have the same styling, by using the program
+//     // count of the larger set of programs
+//     String cardContainerStyles =
+//         programCardViewRenderer.programCardsContainerStyles(
+//             Math.max(
+//                 Math.max(relevantPrograms.unapplied().size(), relevantPrograms.submitted().size()),
+//                 relevantPrograms.inProgress().size()));
+
+//     // will need to handle case where there is a common intake form
+//     content.with(programSectionTitle(messages.at(MessageKey.TITLE_PROGRAMS.getKeyName())));
+
+//     content.with(
+//         programCardViewRenderer.programCardsSection(
+//             request,
+//             messages,
+//             personalInfo,
+//             Optional.of(MessageKey.TITLE_PROGRAMS_ACTIVE_UPDATED),
+//             cardContainerStyles,
+//             applicantId,
```

Comment:
`programCardViewRenderer.programCardsSection` expects `applicantId`. One of the places it uses it is to generate the program details links (if one isn't provided).